src/main.js

```diff
@@ -64,12 +75,31 @@ function switchScreens(closingWindow, openingWindow){
 }
 
 function userInput() {
-  var newPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
-  posterImage.src = newPoster.imageURL;
-  posterTitle.innerHTML = newPoster.title;
-  posterQuote.innerHTML = newPoster.quote;
-  images.push(newPoster.imageURL);
-  titles.push(newPoster.title);
-  quotes.push(newPoster.quote);
-  savedPosters.push(newPoster);
-}
\ No newline at end of file
+  currentPoster = new Poster(userPosterImage.value, userPosterTitle.value, userPosterQuote.value);
+  posterImage.src = currentPoster.imageURL;
+  posterTitle.innerHTML = currentPoster.title;
+  posterQuote.innerHTML = currentPoster.quote;
+  images.push(currentPoster.imageURL);
+  titles.push(currentPoster.title);
+  quotes.push(currentPoster.quote);
+}
+
+function savePoster() {
+  var duplicate = false;
+  for (var i= 0; i < savedPosters.length; i++) {
+    if (savedPosters[i] === currentPoster) {
+      duplicate = true;
+    }
+  }
+  if (!duplicate) {
+      savedPosters.push(currentPoster);
+      console.log(currentPoster.imageURL);
+      var newPoster = `${savedPosterLocation.innerHTML} <article class="poster">
```

Comment:
When saved, the saved poster that shows inside of your `savedPosterLocation` view is very large. It doesn't quite fit what the image on the README shows for this view. Take another look at your stylesheet to see if anything else can help there.